parse/ArithmeticRules.cpp

```diff
@@ -53,7 +53,6 @@ namespace parse::detail {
                         |   tok.NoOp_   [ _c = ValueRef::OpType::NOOP ] // pass-through does nothing, for debug purposes
                         |   tok.abs_    [ _c = ValueRef::OpType::ABS ]
                         |   tok.round_  [ _c = ValueRef::OpType::ROUND_NEAREST ]
-                        |   tok.ceil_   [ _c = ValueRef::OpType::ROUND_UP ]
```

Comment:
this is removing functionality that ValueRefs support... https://github.com/freeorion/freeorion/blob/master/universe/ValueRefs.cpp#L3372

Reply:
They will remain accessible from Python Parser.

Reply:
I'd rather not remove functionality from the Boost parser until the migration to Python is complete.